### ISSUE TITLE
[iOS] UITableView.Appearance.BackgroundColor ignored or overridden for ListView

### ISSUE THREAD

**Issue by jcmanke** [xamarin/Xamarin.Forms#5503]
### Description

In one of my apps, I set `UITableView.Appearance.BackgroundColor = UIColor.Clear;` in the AppDelegate to make ListViews clear, in order to show the BackgroundColor of the page, rather than the default white. In Xamarin.Forms 3.6.0, this no longer works.

### Steps to Reproduce

1. In AppDelegate.cs in the iOS project, add this line: `UITableView.Appearance.BackgroundColor = UIColor.Clear;`
2. Make a ContentPage with a ListView.
3. Set the BackgroundColor of the page.
4. Run with Xamarin.Forms 3.5.0 and then with 3.6.0.

### Expected Behavior

ListViews are transparent, showing the BackgroundColor of the page. 

### Actual Behavior

ListViews are white.

### Basic Information

- Version with issue: 3.6.0.220655
- Last known good version: 
  - Stable: 3.5.0.169047
  - Pre-release: 3.6.0.169048-pre2
- IDE: Visual Studio Enterprise 2017, version 15.9.7
- Platform Target Frameworks:
  - iOS:  12.1

### Screenshots

Before 3.6.0:
![](https://user-images.githubusercontent.com/6132629/54047515-d671da00-419c-11e9-92fc-a40e31db536c.png)

On 3.6.0:
![](https://user-images.githubusercontent.com/6132629/54047530-e5588c80-419c-11e9-9634-01b7154deaef.png)

### Reproduction

[ListViewTransparency.zip](https://github.com/xamarin/Xamarin.Forms/files/2946990/ListViewTransparency.zip)

**Comment by jcmanke** [comment]
Workaround is to add an implicit style in the application resources.

```
<Style TargetType="ListView">
    <Setter Property="BackgroundColor" Value="Transparent" />
</Style>
```

**Comment by kingces95** [comment]
Hm, reproduces as indicated tho I'm not sure we have a policy about twiddling platform properties and expecting Forms to adopt them as defaults.. The work around seems like the way we'd recommend achieving the transparency effect. 

@jcmanke Why twiddle the platform instead of using the workaround that also works x-plat from the get-go?

**Comment by jcmanke** [comment]
From what git blame tells me, we added that line to AppDelegate in 2016, so I'm not completely sure anymore. I think the logic was along the lines of it only being an issue on iOS so we fixed it natively on iOS.

**Comment by samhouts** [comment]
Looks like we need to update our changes made in #4002 to take this into account. Thanks!

**Comment by samhouts** [comment]
closed by #6910
